tcms/core/tests/test_ajax.py

```diff
@@ -74,103 +74,29 @@ def test_update_default_tester_non_existing_user(self):
         response = self.client.post(self.url, {
             'case[]': [case.pk for case in TestCase.objects.filter(plan=self.plan)],
             'what_to_update': 'default_tester',
-            'usernmae': 'user which doesnt exist'
+            'username': 'user which doesnt exist'
         })
 
-        self.assertEqual(HTTPStatus.OK, response.status_code)
+        self.assertEqual(HTTPStatus.NOT_FOUND, response.status_code)
         result = json.loads(str(response.content, encoding=settings.DEFAULT_CHARSET))
         self.assertEqual(result['rc'], 1)
-        self.assertEqual(result['response'], 'Default tester not found!')
+        self.assertEqual(result['response'], 'User user which doesnt exist not found!')
```

Comment:
The error message in the view is wrong. It wraps `"User %s not found" % username` into `_()` for translations. The actual string substitution must be done outside of the translation string.

The same substitution should be done here to pass l10n tests in CI.